ci_framework/hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -13,7 +13,7 @@
         label: "{{ item }}"
 
 - name: Push repositories hook
-  hosts: all
+  hosts: all,!crcs
```

Comment:
I checked the zuul console logs, it is working. I think the excluded host would be `!crc` not `!crcs`. I might be wrong here. Does hostname with plural will work?

Reply:
it will work with the reproducer - there I have a plural for the auto-generated groups. We may add "!crc" actually. Lemme amend it. It will generate a warning, but won't crash.

Reply:
added "!crc" to match the CI, and updated commit message to explain that little hack :)